### ISSUE TITLE
Remove welfare cooldown

### ISSUE THREAD

**Issue by CounterCraftz** [GMDevinity/Flood#190]
Remove welfare cooldown but reduce the amount of money given to $2000

**Comment by Donkie** [comment]
Sigh, again? Cool down is necessary because fucking nobody uses it properly. It's fine as it is atm.

**Comment by CounterCraftz** [comment]
I've soon come to realize that it can be both good and bad
Players who don't actually abuse welfare (and I've seen it) will have to either rely on a team (if they manage their money sufficiently) or team-based event because of how poor they are where it comes to the point where the gamemode is 'unplayable' which drives them away from our servers.

**Comment by Donkie** [comment]
Yeah but then we have to rebalance other things imo. This system should be the help out of a deep spiral where the player has no ability to play again. If that deep spiral happens often, the issue is somewhere else.

**Comment by CounterCraftz** [comment]
Do you believe we've (dawson and i) been too harsh on the economy? I honestly don't know because of the amount of props added and configurations that are being made every so often.

**Comment by Donkie** [comment]
I don't play so I have no idea about the economy, I trust that on you guys. But we should come up with some non linear balancing where it gets more and more difficult to get money. That will both help new players and make it more challenging for veterans

**Comment by Donkie** [comment]
Look into how other games do it maybe

**Comment by CounterCraftz** [comment]
How should we determine/differentiate whether if a player is considered new or not? Do we use level, playtime, the amount of weapons they have (not sure if this can get easily abused) or just everything in general?

Heres what I've come up with (would like @GMDevinity/moderators's feedback)
 
**New Players**
- Props are 7% cheaper 
- Damage to cash conversion & time alive is increased by ; 5%; $1/second
- Welfare cooldown disabled (?)

**(Regular) Players**
- No change

**"Veteran" Players**
- Props are 5% more expensive
- Damage to cash conversion & time alive is decreased by 5%; $1/second
- Winning profit is decreased by 15-20%

- [This thread](http://devinity.org/threads/hard-mode-for-when-flood-is-just-too-easy.3334/) would come in handy. Completely disregard the changes above if the player toggles 'Hard Mode'.

**Comment by Donkie** [comment]
That's some pretty bad artifical limits, something gamers always dislike. It needs to be more subtle gameplay changing things, not basic "fuck you you've played too much here is some tax"

**Comment by Donkie** [comment]
Oh, haven't seen hard mode. That sounds really interesting and could help a lot. The question then becomes again how do we make it easier for new players.

**Comment by CounterCraftz** [comment]
I don't think we're going to find a fix sometimes soon so it would be better if welfare cooldown gets removed (or revert back to the previous economy tweaks; 90% damage to cash conversion and/or $10/sec time alive. Even though this is favoring 'experienced' players by a ton, its better than having players leave just because **they have no cash** which makes it harder for them to progress). Even if I give them cash, retardedly enough, they keep on blowing it it up.

**Comment by DawsonDevinity** [comment]
Yes, please check the document sometime for the sake of being on the same page as everyone.  I've noticed more and more the problem of players going broke, and we need to STOP the BLEEDING.  Averting this would require a number of things to be done which include (but are not limited to) abolishing welfare cooldown, adding more starter weapons (i.e, melee, sidearm, submachine gun), and recalibration of the money gain system.

**Comment by Donkie** [comment]
Okay so the issue here is that players aren't able to play when they run out of money. 

Why can't they play? Because it costs money to build a boat. 

Few ways to solve this:
* Make some props free
* Encourage teamplay, they can enter another team and use their boat
* Make the boat price get deducted -after- the round ends, you buy the props on credit. That way you can build any boat you want, if you manage to not earn the amount required to pay for the boat, you will simply lose money there instead. Your money will never go below 0. This will let anyone build any boat, but they still have to be smart because otherwise they'll never gain money to buy new weapons.

**Comment by DawsonDevinity** [comment]
I actually like the idea of designating some of the more basic props as free.  Counter and I could look into that, for sure.

**Comment by jeffreythe00** [comment]
My suggestion is to log any transactions made after a welfare check is given. We would also update the rules to reflect this change.

IE: 

I get a welfare check, immediately spend it on weapons/tokens/props. This info would be snapshotted and sent to a log file where whitelisted staff can review and decide if the said players are abusing it. In the example below, you can see that it is pretty evident that I have abused welfare.

> Jeffreythe00 has purchased 5 tokens
> Jeffreythe00 has purchased 5 tokens
> Jeffreythe00 has purchased 5 tokens
> Jeffreythe00 has purchased 5 tokens
> Jeffreythe00 has given 2500$ to donkie
> Jeffreythe00 has given 2500$ to donkie
> Jeffreythe00 has given 2500$ to donkie

**Comment by Donkie** [comment]
That would be waaaay too much admin work
